### ISSUE TITLE
random lock on nvda

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#8054]
### Steps to reproduce:
He was using nvda normally.
I opened the nvda add-ons window.
The nvda was mute, and from this moment the nvda did not announce an open key or window.
When I called the narrator, I discovered that the windows were working.
The nvda add-in window was open, but it was not possible to walk with arrows or tab through the window.
Could not close the Add-ons window.
It was necessary to quit nvda through the task manager.
It was the second time that this occurred, however it is not possible to reproduce this error always, it seems to be random.
I was able to save an old log when this error occurred.
### Expected behavior:
> Tell us what should happen.
without locking
### Actual behavior:
> Tell us what happens instead.
errors when using nvda
### System configuration:
NVDA version:
next 14896
NVDA Installed or portable:
instaled
Other information:
> Example: Running in a VM

Windows version:
windows 10 1709 16299.248 64bits
Name and version of other software in use when reproducing the issue:

### Other questions:

Does the issue still occur after restarting your PC?
yes
Have you tried any other versions of NVDA?
> Please list them and the result
no
log attach
[nvda-old.log.txt](https://github.com/nvaccess/nvda/files/1775288/nvda-old.log.txt)
ed

**Comment by leonardder** [comment]
It looks like you have several broken add-ons in your configuration, including lambda, indent tone and emu gui.

**Comment by fernando-jose-silva** [comment]
Thank you very much.
I uninstalled these installed add-ons.
I will use the computer for a while longer and see if the error no longer occurs.

**Comment by leonardder** [comment]
You're welcome. It might be a good idea just to walk through your installed add-ons and clean up whatever you do not need.

**Comment by fernando-jose-silva** [comment]
thank you so much

Enviado do meu iPhone

Em 2 de mar de 2018, à(s) 17:30, Leonard de Ruijter <notifications@github.com> escreveu:

> You're welcome. It might be a good idea just to walk through your installed add-ons and clean up whatever you do not need.
>

**Comment by dkager** [comment]
Could you make the issue title a bit more specific, e.g. NVDA locks when opening the addon manager?

**Comment by fernando-jose-silva** [comment]
nvda crashed twice when opening the addons manager. I am away from the computer, so I have access to it again, change the title of the question.Enviado do meu iPhone

Em 4 de mar de 2018, à(s) 06:32, Davy Kager <notifications@github.com> escreveu:

**Comment by leonardder** [comment]
@fernando-jose-silva: Could you make sure that there's no add-on in your installation causing this? I'm still not convinced that it is NVDA itself causing the issue here. You can either disable/remove all add-ons or run NVDA with add-ons disabled from the restart window (NVDA+Q)

**Comment by fernando-jose-silva** [comment]
thank you, I believe that the problems were caused by add-ons, by disabling the add-ons there was no problem, now I have to re-enable each add-on until I figure out who was the cause, a difficult task because the error is not always reproduced.Enviado do meu iPhone

Em 7 de mar de 2018, à(s) 15:51, Leonard de Ruijter <notifications@github.com> escreveu:

> @fernando-jose-silva: Could you make sure that there's no add-on in your installation causing this? I'm still not convinced that it is NVDA itself causing the issue here. You can either disable/remove all add-ons or run NVDA with add-ons disabled from the restart window (NVDA+Q)
> 
> —

**Comment by leonardder** [comment]
Thanks for investigating. As soon as you know what add-on is the culprit, could you please contact the author or the add-ons mailing list to discuss the matter?

I'm closing this for now, but feel free to request a reopen if you yet believe NVDA itself is in error here.
